pkg/controller/networkpolicy/validate.go

```diff
@@ -749,6 +754,29 @@ func numFieldsSetInStruct(obj interface{}) int {
 	return num
 }
 
+// validateIPBlock validates the CIDR values in the IPBlock.
+func validateIPBlock(ipb *crdv1beta1.IPBlock) (string, bool) {
+	if ipb.CIDR == "" {
+		return "field 'cidr' is required in an ipBlock", false
+	}
```

Comment:
We can keep this, but I assume this is guaranteed by the OpenAPI spec?

Reply:
Yes it should already be guaranteed by openAPI, just trying to keep it on par with https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/networking/validation/validation.go#L219